readme.md

```diff
@@ -72,7 +72,7 @@ Get started with Renovate by checking out our [tutorial](https://github.com/reno
 ### GitHub
 
 We recommend that you use the Mend Renovate App.
-Install the [the Mend Renovate App](https://github.com/apps/renovate) now.
+Install the [Mend Renovate App](https://github.com/apps/renovate) now.
```

Comment:
```suggestion
Install [the Mend Renovate App](https://github.com/apps/renovate) now.
```

Good catch! But we should remove the `the` that's outside the link text. Because our docs site [^1] uses the Material for MkDocs abbreviations feature [^2]. I've set a hover text for these strings [^3]:

```
*[the Mend Renovate App]: This means the Mend-hosted GitHub app.
*[The Mend Renovate App]: This means the Mend-hosted GitHub app.
```

That's why I'm removing the _other_ `the`. 😉 

[^1]: [Renovate docs](https://docs.renovatebot.com/)
[^2]: [Material for MkDocs manual, abbreviations](https://squidfunk.github.io/mkdocs-material/reference/tooltips/#adding-abbreviations)
[^3]: [`abbreviations.md` on Renovate docs source repository](https://github.com/renovatebot/renovatebot.github.io/blob/main/includes/abbreviations.md)